Appearence Enhancements

Pharmaracist opened the pull request.

Pharmaracist commented:
@clsty

clsty commented:
I've noticed this PR and I do think that it's cool, but @end-4 is the one making decision on AGS related things.

Pharmaracist commented:
in minor tweaks i removed the comments

Pharmaracist commented:
![image](https://github.com/user-attachments/assets/72b22930-4437-47ad-a361-d6a57d375f3a)
this is the gowall stuff 
still need abit of work but its working and very fast

Pharmaracist commented:
when i added rofi support i added full rofi setup for just fall back like bluetooth , bookmarks storage area file browser and pdf container its just the keybinds i left it for you to do it since my keybinds are completely different + i need help with foot i really dont know it but it seems to be the one having hot relaod on colorchange and mode change so thats important
